src/Service/AssetsListenerFactory.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Mvc\Service;
+
+use Interop\Container\ContainerInterface;
+use Zend\Mvc\AssetsListener;
+use Zend\ServiceManager\FactoryInterface;
+use Zend\ServiceManager\ServiceLocatorInterface;
+
+class AssetsListenerFactory implements FactoryInterface
+{
+    /**
+     * Create the default dispatch listener.
+     *
+     * @param  ContainerInterface $container
+     * @param  string $name
+     * @param  null|array $options
+     * @return DispatchListener
+     */
+    public function __invoke(ContainerInterface $container, $name, array $options = null)
+    {
+        $listener = new AssetsListener();
```

Comment:
IMO, all required dependencies (e.g. an `AssetManager`) should be pushed in as constructor dependencies, but maybe consistency is a counter-argument here.